Cargo.lock

```diff
@@ -549,7 +549,7 @@ checksum = "c688fc74432808e3eb684cae8830a86be1d66a2bd58e1f248ed0960a590baf6f"
 dependencies = [
  "glob",
  "libc",
- "libloading",
+ "libloading 0.7.4",
```

Comment:
`clang-sys` should only be used in `dev-dependencies` and `build-dependencies` so the duplication of `libloading` is fine and won't affect binary size.